src/gui/shellwidget/shellcontents.cpp

```diff
@@ -22,17 +23,16 @@ bool ShellContents::fromFile(const QString& path)
 	}
 
 	int row = 0;
-	while(!f.atEnd()) {
+	while (!f.atEnd()) {
 		QString line = f.readLine();
-		resize(_rows+1, qMax(_columns, string_width(line)));
+		resize(_rows + 1, qMax(_columns, string_width(line)));
 		put(line, row, 0);
 		row += 1;
 	}
 	return true;
 }
 
-ShellContents::ShellContents(int rows, int columns)
-:_data(0), _rows(rows), _columns(columns)
+ShellContents::ShellContents(int rows, int columns): _data(0), _rows(rows), _columns(columns)
```

Comment:
In general I call initializers in a separate line - but I cant find the correct option for this.

Reply:
It's [Break Constructor Initializers](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) on this page, I don't know which one you prefer. There's also the same with Inheritance List.

ConstructorInitializerAllOnOneLineOrOnePerLine is relevant too, and I'm checking if there's a way to always break even if the line is short enough, but I'm not sure yet I'll find it

Reply:
Another solution for this would be to set `ColumnLimit` to 0, so the formatter never reformats long lines, but it always respect manual breaks